.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,67 +1,52 @@
 ---
-name: Bug report
-about: Create a report to help us improve
```

Comment:
I would like to provide a wording that tells all contributors that their filing a bug actually is a contribution which comes with a certain responsibility and not simply a complaint that something works wrong in their opinion.
Filing a bug is not fingerpointing but helping.

Reply:
Good point, I reformulated the "about" section in 4ed8d970